code/game/machinery/transportpod.dm

```diff
@@ -0,0 +1,121 @@
+/obj/machinery/transportpod
+	name = "Ballistic Transportation Pod"
+	desc = "A fast transit ballistic pod used to get from one place to the next. Batteries not included!"
+	icon = 'icons/obj/structures.dmi'
+	icon_state = "borg_pod_opened"
+
+	density = 1 //thicc
+	anchored = 1
+	use_power = 0
+
+	var/used = 0
+	var/in_transit = 0
+	var/mob/occupant = null
+
+	var/xc = list(137, 209, 163, 110, 95, 60, 129, 201) // List of x values on the map to go to.
+	var/yc = list(134, 99, 169, 120, 96, 122, 189, 219) // List of y values on the map to go to.
+
+	var/limit_x = 3
+	var/limit_y = 3
+
+/obj/machinery/transportpod/process()
+	if(used == 1) // Just to make sure.
+		return
+
+	if(occupant)
+		if(in_transit)
+			var/locNum = rand(0, 7) //pick a random location
+			var/turf/L = locate(xc[locNum], yc[locNum], 1) // Pairs the X and Y to get an actual location.
+			limit_x = xc[locNum]+1
+			limit_y = yc[locNum]+1
+			build()
+			sleep(20) //Give explosion time so the pod itself doesn't go boom
+			src.forceMove(L)
+			playsound(src, pick('sound/effects/Explosion1.ogg', 'sound/effects/Explosion2.ogg', 'sound/effects/Explosion3.ogg', 'sound/effects/Explosion4.ogg'))
```

Comment:
If this makes an actual explosion at the target location it should already be playing these sounds, iirc.

Reply:
Doesn't make an actual explosion.

Reply:
Why is that?

Reply:
I feel like having it be destructive, at least for what I thought it would be used for, would be kind of.. Rude? I could add an option whether you want it to be destructive or not? Right now it just builds some reinforced squares one tile around the drop point.